iot_pkcs11_psa.c

```diff
@@ -52,7 +52,7 @@
 #include "mbedtls/pk_internal.h"
 
 #define PKCS11_PRINT( X )            vLoggingPrintf X
-#define PKCS11_WARNING_PRINT( X )    vLoggingPrintf  X
+#define PKCS11_WARNING_PRINT( X )    /* vLoggingPrintf  X */
```

Comment:
Is this commented out on purpose or just for debugging?

Reply:
it generates too many messages, as a result, AFR IDT fails to parse the test result.

Reply:
Can anything be done about why the warnings are being generated instead of suppressing them?

Reply:
@CharleyChu any update?

Reply:
sorry, I thought you were asking Sherry. This is the warning message that generates a lot during test.

PKCS11_WARNING_PRINT( ( "WARNING: Search parameters other than label are ignored.\r\n" ) );

Are you OK to remove it, or downgrade it by adding "PKCS11_INFO_PRINT()"?

Reply:
I guess why or what's the call path that is causing that warning to get triggered?

Reply:
As commented in https://github.com/Linaro/freertos-pkcs11-psa/blob/master/iot_pkcs11_psa.c#L2101, this port only supports looking up objects by CKA_LABEL and all other search attributes are ignored. This keeps consistent with the mbedtls based implementation https://github.com/aws/amazon-freertos/blob/master/libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c#L2994

Reply:
the caller is xFindObjectWithLabelAndClass() in freertos_plus PKCS11 layer. its search template has both CKA_LABEL and CKA_CLASS https://github.com/aws/amazon-freertos/blob/master/libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c#L300. but here we only support CKA_LABEL, So, the warning message will be print out for every call made from PKCS11 layer.